website/sidebars.ts

```diff
@@ -128,6 +128,7 @@ const sidebars: SidebarsConfig = {
         'advanced/routing',
         'advanced/ssg',
         'advanced/client',
+        'advanced/boosting',
```

Comment:
This is very search/Algolia-specific, and doesn't really belong here.

At best, I'd prefer if we only add a little mention on this page: https://docusaurus.io/docs/search#using-algolia-docsearch

Preferably, it should be a very small section about Algolia boosting, linking to an external documentation written and kept up-to-date by the Algolia team itself.

Reply:
ok, I will talk to the Algolia folks. thanks.